### DISCUSSION TITLE
Mkdocs support

### DISCUSSION THREAD

**Discussion by berot3** [tinacms/tinacms#3194] [Ideas]
I read that Hugo will be supported soon. Now I am wondering if it would be possible to have Tina for mkdocs as well! https://github.com/mkdocs/mkdocs

Especially with the mkdocs-material-theme. 

Of course the markdown files would have to be on git, but besides that, would this theoretically be possible?

The advantage would be huge: allow direct editing and updating of the docs right in the browser!!! 🤯😄

**Comment by jeffsee55** [comment]
This should work fine with our non-React support path https://tina.io/guides/tinacms/non-react-based-ssg/guide/, but you won't be able to preview the changes (ie the sidebar pane with the form in it) because hugo requires a server build on content changes. 

We do have some improvements coming soon for non-react sites, but nothing that would enable live previews.

**Reply by berot3** [reply]
Well that’s something. 
So I could change and update the content in plain markdown and the changes will be saved to git by Tina, which could trigger a rebuild of the docs with my latest changes. 
Sounds not so bad. A live preview would be awesome of course. 

Thanks for the insight!
